iot-hub/Samples/device/DeviceReconnectionSample/readme.md

```diff
@@ -2,6 +2,87 @@
 
 This sample code demonstrates the various connection status changes and connection status change reasons the device client can return, and how to handle them.
 
+### Initialize the client:
+
+```csharp
+// Connection string:
+// Get the device connection string from Azure IoT Portal, or using Azure CLI. 
+// Azure portal - 
+// Navigate to your IoT Hub. From the left pane, under "Explorers", click on "IoT devices".
+// Click and navigate to your device.
+// Copy the connection strings listed (primary and/or secondary).
+// Azure CLI - 
+//  az iot hub device-identity connection-string show --device-id <device_id> [--key-type {primary, secondary}]
+//  --key-type is optional. It defaults to "primary".
+//
+// Transport type:
+// The transport to use to communicate with the IoT Hub. Possible values include Mqtt,
+// Mqtt_WebSocket_Only, Mqtt_Tcp_Only, Amqp, Amqp_WebSocket_Only, Amqp_Tcp_only, and Http1.
+//
+// Pass them to the application using command line parameters (see Parameters.cs).
+
+string connectionString = "<connection_string>";
```

Comment:
I found this confusing. The previous comments talk about passing command-line parameters, but this is showing how a customer might initialize. I expected an example of how to run the sample, but now I see you are showing parts of the sample.

You might add a comment with command-line example, and then another comment to highlight code snippets and what they do.

Reply:
I will remove the note about Parameter.cs from this readme. This readme should show how you would initialize a client in general and the values you need to pass in (in most common cases). You could pass them through command line/ env args/ hardcode them, etc.